apps/sms/views/conversation/js/conversation.js

```diff
@@ -725,18 +729,9 @@ var ConversationView = {
       parametersPromise = Promise.resolve(params);
     }
 
-    return parametersPromise.then((parameters) => {
-      Compose.fromMessage(parameters);
-
-      if (this.recipients.length) {
-        Compose.focus();
-      } else {
-        // If message had subject then it's focused as well causing "interact"
-        // Compose event that sets Recipients.View.isFocusable to false.
-        Recipients.View.isFocusable = true;
-        this.recipients.focus();
-      }
-    });
```

Comment:
much more explicit to use the argument "focusComposer" than trying to "guess" what to do.